app/stac_api/models.py

```diff
@@ -132,6 +146,22 @@ class Meta:
         unique_together = (('rel', 'landing_page'))
 
 
+class ConformancePage(SingletonModel):
+    conforms_to = ArrayField(
+        models.URLField(
+            blank=True,
+            null=True
```

Comment:
Empty string should not be accepted. However empty array could be accepted here but don't have to.